lib/Myriad/Service/Storage.pm

```diff
@@ -53,21 +54,21 @@ method apply_prefix ($k) {
     return $prefix . '.' . $k;
 }
 
-async method get ($k, %args) { return $storage->get($self->apply_prefix($k), %args) }
-async method set ($k, %args) { return $storage->set($self->apply_prefix($k), %args) }
-async method observe ($k, %args) { return $storage->observe($self->apply_prefix($k), %args) }
-async method push ($k, %args) { return $storage->push($self->apply_prefix($k), %args) }
-async method unshift ($k, %args) { return $storage->unshift($self->apply_prefix($k), %args) }
-async method pop ($k, %args) { return $storage->pop($self->apply_prefix($k), %args) }
-async method shift ($k, %args) { return $storage->shift($self->apply_prefix($k), %args) }
-async method hash_set ($k, %args) { return $storage->hash_set($self->apply_prefix($k), %args) }
-async method hash_get ($k, %args) { return $storage->hash_get($self->apply_prefix($k), %args) }
-async method hash_add ($k, %args) { return $storage->hash_add($self->apply_prefix($k), %args) }
-async method hash_keys ($k, %args) { return $storage->hash_keys($self->apply_prefix($k), %args) }
-async method hash_values ($k, %args) { return $storage->hash_values($self->apply_prefix($k), %args) }
-async method hash_exists ($k, %args) { return $storage->hash_exists($self->apply_prefix($k), %args) }
-async method hash_count ($k, %args) { return $storage->hash_count($self->apply_prefix($k), %args) }
-async method hash_as_list ($k, %args) { return $storage->hash_as_list($self->apply_prefix($k), %args) }
+async method get ($k, %args) { return await $storage->get($self->apply_prefix($k)) }
```

Comment:
would suggest just drop the `async` instead

Reply:
I quite didn't get this? most of the storage handling is over the network.